index.js

```diff
@@ -0,0 +1,196 @@
+export const attention = {
+  base: 'border-2 relative',
+  tooltip: 'bg-gray-700 border-gray-700 text-white rounded-4 py-6 px-8',
+  callout: 'bg-green-100 border-green-400 py-8 px-16 rounded-8',
+  popover: 'bg-white border-white rounded-8 p-16 filter drop-shadow-20',
+  arrowBase: 'absolute h-14 w-14 border-2 border-b-0 border-r-0 transform',
+  arrowTooltip: 'bg-gray-700 border-gray-700',
+  arrowCallout: 'bg-green-100 border-green-400',
+  arrowPopover: 'bg-white border-white'
+}
+
+export const pageIndicator = {
+    wrapper: 'flex space-x-8 p-8',
+    dot: 'h-8 w-8 rounded-full',
+    inactive: 'bg-bluegray-300',
+    active: 'bg-blue-600',
+};
+
+export const ribbon = {
+    base: 'py-4 px-8 border rounded-4 inline-flex last-child:mb-0',
+    info: 'border-aqua-50 bg-aqua-50',
+    success: 'border-green-100 bg-green-100',
+    warning: 'border-yellow-100 bg-yellow-100',
+    error: 'border-red-100 bg-red-100',
+    disabled: 'border-bluegray-300 bg-bluegray-300',
+    sponsored: 'border-aqua-200 bg-aqua-200',
+    neutral: 'border-gray-100 bg-gray-100',
+    outlined: 'border-bluegray-300 bg-white',
+};
+
+export const slider = {
+    wrapper: 'f-slider relative w-full h-44 py-2',
```

Comment:
All of the `f-` prefixes should be `w-`, I guess? But not sure if we should do that now or save it for later? 🤷

Reply:
Those classes might be able to 🔥 now that we have better utility class support. I think wait-and-see is fine here?

Reply:
Yea, let's leave it for later. A separate PR is also better to isolate the changes for the prefixes, I **_think_**